interop/test_utils.go

```diff
@@ -540,6 +579,10 @@ func (s *testServer) FullDuplexCall(stream testpb.TestService_FullDuplexCallServ
 		if err != nil {
 			return err
 		}
+		resp := in.GetResponseStatus()
```

Comment:
We don't.  It's only being called once.  (The call right below this one is to GetResponseParameters(), not GetResponseStatus().)